gobo/awesome/sound.lua

```diff
@@ -145,7 +145,9 @@ local function update_icon(widget, state)
    widget:set_image(image)
 end
 
-function sound.new()
+function sound.new(options)
+	 local mixer = options.mixer or "ncpamixer"
```

Comment:
```suggestion
	 local mixer = options and options.mixer or "ncpamixer"
```

Otherwise, this will crash if `options` is not passed.

I think this is missing a `terminal` option too, see below